include/control_toolbox/pid.hpp

```diff
@@ -196,7 +197,25 @@ class CONTROL_TOOLBOX_PUBLIC Pid
    *
    * \note New gains are not applied if i_min_ > i_max_
    */
-  void initPid(double p, double i, double d, double i_max, double i_min, bool antiwindup = false);
+  void init_pid(double p, double i, double d, double i_max, double i_min, bool antiwindup = false);
```

Comment:
As we are here, can we change it to simple initialize? I feel it is kind of redundant in the naming. If you think we should go this way, then let's do it

```suggestion
  void initialize(double p, double i, double d, double i_max, double i_min, bool antiwindup = false);
```

Reply:
you are right